Cargo.toml

```diff
@@ -41,6 +41,10 @@ default-features = false
 version = "0.3"
 default-features = false
 
+[dependencies.sprs]
+git = "https://github.com/pmarks/sprs.git"
```

Comment:
This is required for the Dot impl in @pmarks branch, which itself is dependent on https://github.com/rust-lang/rust/issues/55437

Reply:
We need to create nightly CI